### DISCUSSION TITLE
Recommended Setup for new miners?

### DISCUSSION THREAD

**Discussion by davidawad** [filecoin-project/lotus#5489] [Storage Provider] [answered]
I'm a small miner looking to get started and learn whether filecoin mining makes sense for me as I'm long on this concept. 

I found an example recommended setup but this seems like a big investment as a small miner ($4000). 
https://pcpartpicker.com/user/tperson/saved/H2BskL

Is there any way to get started a little smaller time on something cheaper (maybe raspberry pi + a hard drive)? 

Thanks

**Comment by Fatman13** [comment, marked as answer]
It is unlikely that you can mine filecoin with raspberry pi for now. There are lots of threads discussing hardware setups in slack. For example, this [one](https://filecoinproject.slack.com/archives/CG6P10H9T/p1604943788198700) or this [one](https://filecoinproject.slack.com/archives/CG6P10H9T/p1604948027201600?thread_ts=1604943788.198700&cid=CG6P10H9T).

**Reply by davidawad** [reply]
got it, so I'll dig into these threads. 

As a separate question, if slack isn't being used anymore, is github now the only place for discussion on filecoin mining? 
thanks

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
the slack has 5000+ users and is very much active

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
join #fil-mainnet #fil-help #fil-lotus for mining questions / discussions

**Reply by davidawad** [reply]
awesome, I'll jump back in there then thanks!

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
also see this for the general requirements the process steps have

https://docs.filecoin.io/mine/hardware-requirements/

with 4k you are at the end of the spectrum, you might be able to save a few hundred bucks on hardware but those numbers are easily topped by the money you have to invest in FIL
